requirements.txt

```diff
@@ -4,8 +4,7 @@ Pillow
 cython
 matplotlib
 scikit-image
-tensorflow>=1.3.0
-keras>=2.0.8
+tensorflow == 2.6.0
```

Comment:
I got something working on our data with:
- python 3.9.12
- keras 2.9.0
- tensorflow 2.9.1
Although I didn't try to run the "samples" proposed.